terraso_backend/apps/graphql/schema/commons.py

```diff
@@ -249,3 +250,21 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
 
         result_kwargs = {from_camel_to_snake_case(cls.model_class.__name__): model_instance}
         return cls(**result_kwargs)
+
+    @classmethod
+    def not_allowed(cls):
+        raise super().not_allowed(cls.model_class, MutationTypes.DELETE)
+
+    @classmethod
+    def get_or_throw(cls, field_name, id_):
+        try:
+            return cls.model_class.objects.get(id=id_)
+        except cls.model_class.DoesNotExist:
+            raise GraphQLNotFoundException(
+                field_name=field_name, model_name=cls.model_class.__name__
+            )
```

Comment:
question: can we avoid duplicating the other `get_or_throw()` methods?

Reply:
I edited the files to clean this up, now there is only one `get_or_throw`